release notes/v0.42.0.md

```diff
@@ -0,0 +1,151 @@
+k6 v0.42.0 is here and it brings a few exciting updates! :tada:
+
+## New Features
+
+### Experimental Prometheus Remote Write Output [#2784](https://github.com/grafana/k6/pull/2784)
+
+Added a new builtin Output to any [Prometheus Remote Write](https://docs.google.com/document/d/1LPhVRSFkGNSuU1fBd81ulhsCPR4hkSZyyBj1SZ8fWOM/edit) implementations (e.g. [Prometheus](https://prometheus.io/docs/prometheus/latest/feature_flags/#remote-write-receiver), [Mimir](https://grafana.com/docs/mimir/latest/operators-guide/reference-http-api/#remote-write)). This is an experimental API and it could change in the future introducing breaking changes.
+
+Below is a simple example of using the Output in a `k6 run` command using the defaults options like the Remote Write server URL (http://localhost:9090/api/v1/write):
```

Comment:
```suggestion
Below is a simple example of using the new output in a `k6 run` command using the defaults options like the Remote Write server URL (http://localhost:9090/api/v1/write):
```

It seems strange to me to have the `O` uppercase and calling it "the output" makes it sound as it is the only output to me or like if it's something we didn't have anything now.